CorsixTH/Lua/dialogs/queue_dialog.lua

```diff
@@ -176,16 +176,17 @@ function UIQueue:onMouseUp(button, x, y)
       return
     end
 
+    local queue_reported_humans = queue:size() - num_patients
```

Comment:
Funny name for this variable - not clear what it represents. (I assume from the context that it's people who need to use the door that are not part of the patient queue, like staff that need to enter the room.)

Reply:
Yeah a little funky, in light of this I've done a refactor and I'll push that out once I tested it, as its a little more readable also. This currently has this changed to first_patient_index